src/Sparrow/Hashing.Streamed.cs

```diff
@@ -47,26 +49,47 @@ public static void Process(ref XXHash64Context context, byte* buffer, int size)
 
                     if (context.BufferSize + size >= Alignment)
                     {
-                        ulong v1 = context.Current.V1;
-                        ulong v2 = context.Current.V2;
-                        ulong v3 = context.Current.V3;
-                        ulong v4 = context.Current.V4;
-
+#if NET8_0_OR_GREATER
+                    if (Avx512F.VL.IsSupported)
```

Comment:
Use `AdvInstructionSet.IsSupportedAvx512Basic` instead of the `Avx512F.VL.IsSupported` because we have support in the testing environment to control those.

Reply:
Will do.